### ISSUE TITLE
Catchable Fatal Error with JMSPaymentCoreBundle

### ISSUE THREAD

**Issue by qdequippe** [E-vence/SoftDeleteableListenerExtensionBundle#3]
Hi,

I have an error using this bundle, please find the error below
```
Catchable Fatal Error: Argument 1 passed to JMS\\Payment\\CoreBundle\\Entity\\Credit::__construct() 
must implement interface JMS\\Payment\\CoreBundle\\Model\\PaymentInstructionInterface, none 
given, called in /Users/dequippe/Sites/studeal-rest/vendor/evence/soft-deleteable-extension-
bundle/Evence/Bundle/SoftDeleteableExtensionBundle/EventListener/SoftDeleteListener.php on line
48 and defined
```
This error occurs when I try to delete an entity which is softDeletable.
I tried to update JMSPaymentCoreBundle but I have the same problem.

Do you have any idea what can cause this problem?

Please find my dependencies versions:
Symfony 2.8.2
JMSPaymentCoreBundle 1.3.0

**Comment by qtsd** [comment]
Seems fixed by the last commit, but no released.
Use dev-master in your composer.json, or wait for a release tag.

**Comment by qdequippe** [comment]
Thanks @qtsd problem fixed!

**Comment by RubenHarms** [comment]
Last commit is now released under 1.2!
